File: Install-OpenCV-4-on-Ubuntu-from-source/sampleCode.py
import cv2 as cv

# Print OpenCV Version
print(cv.__version__)

# Read image
image = cv.imread("boy.jpg", 1)

# Convert image to grayscale
image = cv.cvtColor(image, cv.COLOR_BGR2GRAY)

# Display image
cv.imshow("Display", image)
cv.waitKey(0)

# Save grayscale image
cv.imwrite("boyGray.jpg",image)

File: Install-OpenCV-4-on-Ubuntu-from-source/sampleCode.cpp
#include <opencv2/opencv.hpp>
#include <iostream>

using namespace std;
using namespace cv;

int main(int argc, char **argv)
{
    // Print OpenCV Version
    cout << CV_VERSION << endl;
    
    // Read image
    Mat image = imread("boy.jpg", 1);
    
    // Convert image to grayscale
    cvtColor(image, image, COLOR_BGR2GRAY);
    
    // Display image
    imshow("Display", image);
    waitKey(0);
    
    // Save grayscale image
    imwrite("boyGray.jpg",image);
    
    return 0;
}
